src/Dynamic/Includes/cards/taskcard.pug

```diff
@@ -0,0 +1,27 @@
+- var statusClass = ''
+case task.lastStatus
+    when 'Successful': - statusClass= 'has-text-success'
+    when 'Failed': - statusClass= 'has-text-danger'
```

Comment:
This feels super unclean to be mixing JS & Pug specific syntax.
I'd like to avoid using Pug syntax when pure JS can be used instead.